docs/src/content/docs/fr/components/asides.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Encarts
+description: Apprenez à utiliser les encarts dans Starlight pour afficher des informations secondaires à côté du contenu principal d'une page.
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Pour afficher des informations secondaires à côté du contenu principal d'une page, utilisez le composant `<Aside>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Aside slot="aperçu">
+Incluez des informations supplémentaires non essentielles dans un `<Aside>`.
```

Comment:
```suggestion
Incluez des informations supplémentaires non essentielles avec le composant `<Aside>`.
```

I think we should specify "composant" here as "aside" is not a word in French contrary to the English version which can easily omit it.